src/core/bittorrent/torrenthandle.cpp

```diff
@@ -1055,12 +1055,12 @@ int TorrentHandle::downloadPayloadRate() const
     return m_nativeStatus.download_payload_rate;
 }
 
-int TorrentHandle::totalPayloadUpload() const
+qlonglong TorrentHandle::totalPayloadUpload() const
```

Comment:
Why the hate of qlonglong :)  I'm with you all I would have used qint64 but qlonglong was already there in totalDownload() and totalUpload() and that's not my code ;)  So I tried to follow the code style that was already there.